packages/gatsby-theme-newrelic/src/components/Banner.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const Banner = ({ children, color }) => {
+  const [visible, updateVisible] = useState(true);
+
+  if (!visible) return null;
+
+  return (
+    <div
+      css={css`
+        height: 90px;
+        background-color: ${color || '--tertiary-background-color'};
```

Comment:
You'll need to use `var(...)` here to ensure the background color uses the CSS variable otherwise this won't work.